packages/web/src/pages/sign-on-page/AudiusValues.tsx

```diff
@@ -17,21 +16,22 @@ const messages = {
   adFree: 'Ad-Free, Offline Listening'
 }
 
-type AudiusValueProps = { icon: IconComponent; text: string }
+type AudiusValueProps = { icon: IconComponent; text: string; dynamic?: boolean }
 
 /**
  * Each individual audius value text + icon row
  */
 const AudiusValue = (props: AudiusValueProps) => {
   const { icon: Icon, text } = props
-  const { isDesktop } = useMedia()
+  const isSmallDesktop = useMediaQuery('(min-width: 1363px)')
```

Comment:
🤔 In the future I feel like we should put standardized device breakpoints in the harmony theme so we could do something like this
```
const { breakpoints } = useTheme
useMediaQuery(`(min-width: ${breakpoints.smallDesktop.max})`)
```
and get a little less magicky with the string

Reply:
agreed this is really hacky, and would love to have the breakpoints established, we kinda have that with our own useMedia. Im going to chat with sammie about the audius values text and what we should do, and ideally we just use established useMedia values